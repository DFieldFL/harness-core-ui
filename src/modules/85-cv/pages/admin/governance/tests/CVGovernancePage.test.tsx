import React from 'react'
import { waitFor, render } from '@testing-library/react'
import CVGovernancePage from '../CVGovernancePage'

describe('Unit tests for CVGovernancePage', () => {
  test('Ensure page renders', async () => {
    const { container } = render(<CVGovernancePage />)
    await waitFor(() => expect(container.querySelector('[class*="pageBody"]')).not.toBeNull())
    expect(container).toMatchSnapshot()
  })
})
