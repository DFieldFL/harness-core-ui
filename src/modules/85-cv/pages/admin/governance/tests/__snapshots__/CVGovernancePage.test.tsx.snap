// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for CVGovernancePage Ensure page renders 1`] = `
<div>
  <div
    class="Layout--horizontal StyledProps--main container standard StyledProps--flex StyledProps--padding-left-large StyledProps--padding-right-large"
  >
    <h2
      class="StyledProps--font StyledProps--font-h2 StyledProps--main StyledProps--color StyledProps--color-grey800"
      level="2"
    >
      GOVERNANCE SETTINGS
    </h2>
    <div
      class="StyledProps--font StyledProps--main"
    >
      <button
        class="bp3-button StyledProps--font Button--button StyledProps--main"
        type="button"
      >
        <span
          class="bp3-button-text"
        >
          + new item
        </span>
      </button>
    </div>
  </div>
  <div
    class="StyledProps--font StyledProps--main pageBody"
  >
    <div
      class="StyledProps--font StyledProps--main StyledProps--flex StyledProps--flex-align-center-center"
      style="width: 100%; height: 100%;"
    >
      <div
        class="Layout--vertical Layout--layout-spacing-medium StyledProps--main"
        style="align-items: center; margin-top: -48px; width: 470px;"
      >
        <span
          class="bp3-icon StyledProps--main StyledProps--color StyledProps--color-grey400"
        >
          <svg
            height="48"
            viewBox="0 0 27 27"
            width="48"
          >
            <filter
              id="nav-dashboard_svg__a"
            >
              <fecolormatrix
                in="SourceGraphic"
                values="0 0 0 0 0.453512 0 0 0 0 0.490089 0 0 0 0 0.533316 0 0 0 1.000000 0"
              />
            </filter>
            <filter
              id="nav-dashboard_svg__b"
            >
              <fecolormatrix
                in="SourceGraphic"
                values="0 0 0 0 0.453512 0 0 0 0 0.490089 0 0 0 0 0.533316 0 0 0 1.000000 0"
              />
            </filter>
            <g
              fill="none"
              fill-rule="evenodd"
              filter="url(#nav-dashboard_svg__a)"
              transform="translate(-100 -231)"
            >
              <g
                filter="url(#nav-dashboard_svg__b)"
                transform="translate(100 231)"
              >
                <rect
                  height="26"
                  opacity="0.5"
                  rx="5"
                  stroke="#cdcaca"
                  width="26"
                  x="0.5"
                  y="0.5"
                />
                <path
                  d="M13.656 8H19v4h-5.344zm0 12v-6.656H19V20zM7 20v-4h5.344v4zm0-5.344V8h5.344v6.656z"
                  fill="#747d88"
                  fill-rule="nonzero"
                />
              </g>
            </g>
          </svg>
        </span>
        <h2
          class="StyledProps--font StyledProps--font-h2 StyledProps--main StyledProps--font-align-center StyledProps--color StyledProps--color-grey500"
          level="2"
        >
          No data found. Click the button below to create a new item.
        </h2>
        <button
          class="bp3-button StyledProps--font Button--button StyledProps--main StyledProps--intent-primary"
          type="button"
        >
          <span
            class="bp3-button-text"
          >
            + new item
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
